src/components/Menu/Menu.tsx

```diff
@@ -88,6 +96,7 @@ class Menu extends AutoControlledComponent<Extendable<MenuProps>, any> {
     secondary: customPropTypes.every([customPropTypes.disallow(['primary']), PropTypes.bool]),
     underlined: PropTypes.bool,
     vertical: PropTypes.bool,
+    submenu: PropTypes.bool,
```

Comment:
here is another clear example - lets just take a look on the following code:

<XXX submenu={...} />

given this code, what we might expect to be passed as a `submenu`'s value? actually, I would expect items collection. This is what we have for `MenuItem`'s `menu`, but here - this is a boolean flag :)

Reply:
Totally agree, for consistency I will leave it as this, and will create issue for renaming all boolean props to have the `is` prefix. Thanks for pointing this out.